src/sentry/demo/data_population.py

```diff
@@ -1333,8 +1333,8 @@ def send_aggr_session(self, dsn, time, mobile, version, num_versions, seen_versi
             exited = random.choices([1, 2, 3, 4], k=1, weights=[10, 5, 3, 1])[0]
             outcome = random.random()
 
-            if outcome <= threshold:
-                crashed = int(random.uniform(1, 11))
+            if outcome > threshold:
+                crashed = int(random.uniform(1, 10))
```

Comment:
@roggenkemper Is it possible that `exited` will have a greater value than `crashed`? I think we should make it so that doesn't happen. Also, `threshold` should correspond to an error rate. I don't think that's happening since `random.uniform(1, 10)` could return `10`. I'll approve this for now but next week, we should refactor this so `threshold` more closely maps to an actual error rate.